packages/css-library/src/stylesheets/elements.scss

```diff
@@ -5,3 +5,197 @@
 // p {
 //    @include @usa-paragraph
 // }
+
+@use "../../dist/tokens/scss/variables" as *;
+
+$color-link-default-hover: rgba($color-black, 0.05);
+
+/* 
+ Styles that are specific to VA.gov using CSS type selectors (elements/tags).
+*/
+
+.feature {
+  background: $color-primary-alt-lightest;
+  padding: 1em;
+  clear: both;
+  margin: 1.5em 0 1.5em 0;
+
+  p {
+    color: inherit;
+    font: inherit;
+    margin-bottom: 0;
+    margin-top: 0;
+    padding-bottom: 0.5em;
+  }
+
+  h2 {
+    margin-top: 0;
+  }
+
+  h3 {
+    margin-top: 0;
+  }
+  h4 {
+    margin-top: 0;
+  }
+  ul {
+    margin: 0 0 0.5em 1.5em;
+    padding: 0;
+    li {
+      list-style: square;
+      margin: 0;
+    }
+  }
+}
+
+.va-quicklinks {
```

Comment:
This is being used in content-build in one place. Is it possible to add this CSS somewhere for _just_ content-build? If so, then we can remove it from the CSS Library.

<img width="445" alt="Screenshot 2023-12-11 at 2 55 59 PM" src="https://github.com/department-of-veterans-affairs/component-library/assets/872479/7ea97395-d419-4b4c-bbd5-ce093c1b6193">